src/frontend/apps/desk/src/features/header/Header.tsx

```diff
@@ -69,28 +70,13 @@ export const Header = () => {
               padding: 0;
             }
           `}
-          $gap="5rem"
+          $gap="1rem"
           $justify="flex-end"
           $direction="row"
         >
-          <Box $align="center" $direction="row">
-            <LanguagePicker />
-          </Box>
-          <Box $direction="row" $align="center">
-            <Box $direction="row" $align="center" $gap="1rem">
-              <Text $weight="bold" $theme="primary">
-                John Doe
-              </Text>
-              <Image
-                width={58}
-                height={58}
-                priority
-                src={IconMyAccount}
-                alt={t(`Profile picture`)}
-              />
-            </Box>
-            <ApplicationsMenu />
-          </Box>
```

Comment:
Small css prob:
![image](https://github.com/numerique-gouv/people/assets/25994652/ec3263e9-86f0-4090-b17a-3e6fa35709af)

You can remove that to fix it:
https://github.com/numerique-gouv/people/blob/86630bb898a9362ab0a9c9fe78a3f52995035fa7/src/frontend/apps/desk/src/features/header/Header.tsx#L68-L72

Reply:
Noted! The given CSS from the gaufre website should be more robust and you should not have to handle custom fixes like that. Working on that next week.

Reply:
wrong rebase !